packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -2,7 +2,7 @@ import { Theme } from '@fluentui-react-native/framework';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { ButtonTokens } from '.';
 
-export const buttonStates: (keyof ButtonTokens)[] = ['fluid', 'primary', 'ghost', 'hovered', 'focused', 'pressed', 'disabled'];
+export const buttonStates: (keyof ButtonTokens)[] = ['fab', 'fluid', 'primary', 'ghost', 'hovered', 'focused', 'pressed', 'disabled'];
```

Comment:
The 'fab' acronym seems like it will be hard for subsequent contributors to figure out.  Is there something longer but more descriptive/self-evident that we could call it?

Reply:
probably just `floating` , since fab stands for floating-action-button. It's an industry term though

Reply:
Yeah, we'd also want to make sure clients understand what `fab` means. If we feel like `floating` is more descriptive and easier to understand for both devs and clients, we can switch to `floating`.

Reply:
But if `fab` is common in the industry and people coming into contact with it would know the term, we can also stick with `fab`.

Reply:
Yeah I vote to keep it unless someone else complains